pkg/sensors/exec/config/config_windows.go

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package config
+
+// ExecObj returns the exec object based on the kernel version
+func ExecObj() string {
+	return "process_monitor.sys"
```

Comment:
How is this file distributed?

Reply:
This change is removed from the PR. This will be distributed with the tetragon msi.

Reply:
hum, not sure what that means, how can we see that code?